woocommerce/woocommerce-android
# Free Trial: Add missing redesign for More menu header

Summary
==========
Fix issue #8810 by adjusting the More Menu screen header to follow the redesign introduced in the view body. The Site picker should now be displayed whenever we hit the header as a button.

Screenshots
==========
| Light Mode  | Dark Mode |
| ------------- | ------------- |
| ![Screenshot_20230418_005638](https://user-images.githubusercontent.com/5920403/232667393-76a00c45-6651-4625-8c53-adb535557ca0.png) | ![Screenshot_20230418_005646](https://user-images.githubusercontent.com/5920403/232667402-e91eaf36-a4e4-4399-b40e-5ebcfb2742fe.png) |

How to Test
==========
1. Open the app and go to the more menu view
2. Make sure the UI is correctly following the color pattern of both Light and Dark mode
3. Make sure the Site picker is accessible as expected
4. Make sure the UI is correctly displayed in landscape mode

Update release notes:
